### ISSUE TITLE
v0.6.0-rc2 User Folder Share Still Not Working on Linux

### ISSUE THREAD

**Issue by atomi** [docker/machine#2980]
Tested on v0.6.0-rc2 shared user folder still not working on Linux using the local Virtualbox driver.
I've gone through some of the older issues for this and it appears a fix should have been merged by now.
Maybe I missed something?

**Comment by nathanleclaire** [comment]
Hm, it should work.  Maybe you have an older version of `boot2docker.iso` in the cache?  If it's not a recent one it won't work.

**Comment by atomi** [comment]
If I delete ~/.docker directory v0.6.0-rc2 downloads boot2docker from https://github.com/boot2docker/boot2docker/releases/download/v1.9.1/boot2docker.iso

```bash
$docker-machine create -d virtualbox ehlo
$docker-machine ssh ehlo ls /home
docker
```
There is no /home/atomi folder :(

**Comment by nathanleclaire** [comment]
OK thanks.  Do you see the share as created in the VirtualBox GUI?

**Comment by nathanleclaire** [comment]
cc @chantra Can you please take a look?

**Comment by atomi** [comment]
Here's the Shared Folders settings:
![screenshot 2016-02-02 16 05 08](https://cloud.githubusercontent.com/assets/83137/12768748/c3ea0976-c9c6-11e5-83c0-723fdf367349.png)

**Comment by nathanleclaire** [comment]
Argh, I thought the relevant modification had been merged into boot2docker, but it looks like it hasn't yet: https://github.com/boot2docker/boot2docker/pull/1086

**Comment by nathanleclaire** [comment]
In the meanwhile, you could build your own `boot2docker.iso` using that branch and use `--virtualbox-boot2docker-url`, then shared folders should work like a charm on Linux.

**Comment by atomi** [comment]
Thanks @nathanleclaire - I'll build boot2docker locally for the time being as I doubt the PR will make it to 1.10.

**Comment by chantra** [comment]
@nathanleclaire @atomi yeah, until boot2docker/boot2docker#1086 get pulled in, this will not work.
